src/browser/components/SelectCertificateModal.jsx

```diff
@@ -51,23 +51,19 @@ export default class SelectCertificateModal extends React.Component {
     const subjectShort = this.maxSize(cert.subject.commonName, CELL_SIZE);
```

Comment:
What about using css to truncate the text and add the ellipsis instead? Calculating text width using a variable-width typeface can be unreliable.